doc/changes/latest.rst

```diff
@@ -14,7 +14,7 @@ NEW
 
 Fixes
 -----
-
+- Fix _NEUROVAULT_BASE_URL and _NEUROSYNTH_FETCH_WORDS_URL in ``nilearn/datasets/neurovault.py``(http --> https)
```

Comment:
```suggestion
- Fix ``_NEUROVAULT_BASE_URL`` and ``_NEUROSYNTH_FETCH_WORDS_URL`` in ``nilearn/datasets/neurovault.py`` by using https instead of http (:gh:`3281` by `Manon Pietrantoni`_).
```